integration/simulation/network/network_utils.go

```diff
@@ -101,7 +101,7 @@ func createInMemTenNode(
 		MaxRollupSize:             1024 * 128,
 		BaseFee:                   big.NewInt(1), // todo @siliev:: fix test transaction builders so this can be different
 		GasLocalExecutionCapFlag:  params.MaxGasLimit / 2,
-		GasBatchExecutionLimit:    params.MaxGasLimit / 2,
+		GasBatchExecutionLimit:    30_000_000,
```

Comment:
This is to match up sims to testnet?

Reply:
yes. It needs to be consolidated at some point, because these configs are all over the place.
No clue why we're not using the defaults everywhere.